.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"
+      - name: Log into ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Get bot token from Parameter Store
+        uses: marvinpinto/action-inject-ssm-secrets@latest
+        with:
+          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v2
+        id: semantic
+        env:
+          GITHUB_TOKEN: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+      - name: Install yq if needed
+        if: steps.semantic.outputs.new_release_published == 'true' || github.event.inputs.environments != ''
+        run: |
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CC86BB64
+          sudo add-apt-repository ppa:rmescandon/yq
+          sudo apt update
+          sudo apt install yq -y
+      - name: Set component versions as environment variables
+        run: |
+          for COMPONENT in $(cat versions.json | jq -r '.components | .[] | .name + "=" + (.version)')
+          do
+            echo "$COMPONENT" >> $GITHUB_ENV
+          done
+      - name: Check out Manifest Repo
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/vsp-infra-application-manifests
+          token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          fetch-depth: 1
+          path: vsp-infra-application-manifests
+      - name: Update image name in Manifest repo
+        if: steps.semantic.outputs.new_release_published == 'true' || github.event.inputs.environments != ''
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: dsva/platform-console
+        run: |
+          cd vsp-infra-application-manifests/apps/vsp-tools-backend/platform-console
+          # Update platform-console image
+          envs=( ${{ needs.prepare-values.outputs.environments }} )
+          for env in ${envs[*]};
+          do
+            yq e -i '.spec.template.spec.containers.[0].image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/platform-console:ruby-${{ env.ruby }}-${{ github.sha }}"' $env/deployment.yml
+          done
+          git diff
+      - name: Add and Commit file
+        if: steps.semantic.outputs.new_release_published == 'true' || github.event.inputs.environments != ''
+        uses: EndBug/add-and-commit@v7
+        with:
+          branch: main
+          add: '*'
+          cwd: vsp-infra-application-manifests/apps/vsp-tools-backend/platform-console
+          author_name: va-vsp-bot
+          author_email: devops@va.gov
+          message: 'auto update platform-console ${{ needs.prepare-values.outputs.environments }} images'
```

Comment:
Would it make sense to add the commit range or release version in this message?

Reply:
> Do you have any thoughts on how we could clean this up for VFS apps in the future to use? Maybe we could extra a GH action that they could import and pass a few variables to?

Yes good idea... I believe they would only need the cluster env, and a few dir names from the manifests repo that they could pass in